frontend-ssr/pages/api/notifications/mockAsRead.ts

```diff
@@ -0,0 +1,56 @@
+import { uuid } from '~/services/Utils';
```

Comment:
We shouldn't need this file. We don't return anything when marking a notification as read.